staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -114,6 +115,13 @@ func (a customResourceStrategy) PrepareForCreate(ctx context.Context, obj runtim
 	}
 
 	accessor, _ := meta.Accessor(obj)
+	if len(accessor.GetAnnotations()[genericapirequest.AnnotationKey]) > 0 {
```

Comment:
idiomatic: `_, found := accessor.GetAnnotations()[genericapirequest.AnnotationKey]; found`

Reply:
Funny, I told Joaquim to just do `if annotations[key] != ""` the other day. I'd call this one personal preference maybe.

Reply:
if the language was more strict we wouldn't have to discuss at least 3 different ways of reading a value from a map :)